### DISCUSSION TITLE
Add WebGPU in Godot for WEB performance

### DISCUSSION THREAD

**Discussion by ElectroidDes** [godotengine/godot-proposals#8892] [Platforms]
I know that similar topics have already been discussed, but I’m interested in asking again why, to improve the performance of Godot on the Web, not implement something ready-made into Godot, for example: **Orillusion - Web3D Rendering Engine based on WebGPU**

[https://www.youtube.com/watch?v=brNav4rDPHU&t=2s](https://github.com/godotengine/godot-proposals/issues/url)

[https://github.com/Orillusion/orillusion](https://github.com/godotengine/godot-proposals/issues/url)
[https://orillusion.github.io/orillusion-webgpu-samples/#gpuParticles](https://github.com/godotengine/godot-proposals/issues/url)
[https://www.orillusion.com/en/example/animation/Skeleton3.html](https://github.com/godotengine/godot-proposals/issues/url)

If Godot supported WebGPU, it would currently be the only universal game engine that would support this technology.

**Comment by AThousandShips** [comment]
Duplicate of https://github.com/godotengine/godot-proposals/issues/6646

This is already being discussed and we want this gesture, we're asking for people to implement it 🙂

I'm the future don't open duplicate things when you know it's already being discussed
